scripts/variantreport_p_sample.rmd

```diff
@@ -288,15 +272,32 @@ sig_mutations.df <- as.data.frame(qpcR:::cbind.na(b117=b117$AA_mut, b1351=b1351$
 sig_mutations.df <- sig_mutations.df %>%
                           tidyr::pivot_longer(everything()) %>%
                             tidyr::drop_na(.,value)
+
+# deduplicate dataframe
+# fixme: I was not sure, how I can include this in my function but should be possible, will come back to it later
+dupes <- duplicated (sig_mutations.df$value, fromLast=TRUE)
+allDupes <- dupes | duplicated (sig_mutations.df$value, fromLast=FALSE)
+
```

Comment:
this looks like a partial copy of the `dedupe` procedure.

Could this be merged with `dedupe`?  Or should `dedupe` be split into two parts to get the list of duplicate columns with one procedure and deduplicate the data frame in another procedure?

Reply:
I think that partially goes with my comment from above. 

But yes - I think splitting dedupe would be a good idea.